src/views/TrackPositionAnimation/TrackPositionAnimationView.tsx

```diff
@@ -171,8 +171,15 @@ const TrackPositionAnimationView: FunctionComponent<TrackPositionAnimationProps>
 
     const { focusTime, setTimeFocus } = useTimeFocus()  // state imported from recording context
     const findNearestTime = useTimeLookupFn(animationState)
-    useEffect(() => matchFrameToFocus(focusTime, findNearestTime, animationState, animationStateDispatch), [focusTime])
-    useEffect(() => matchFocusToFrame(animationState, focusTime, setTimeFocus), [animationState.isPlaying])
+    const animationCurrentTime = animationState?.frameData[animationState?.currentFrameIndex]?.timestamp
+    useEffect(() => {
+        matchFrameToFocus(focusTime, findNearestTime, animationStateDispatch)
+    }, [focusTime, findNearestTime])
+    
+    useEffect(() => {
+        if (animationState.isPlaying) return
+        matchFocusToFrame(animationCurrentTime, setTimeFocus)
+    }, [animationCurrentTime, setTimeFocus, animationState.isPlaying])
```

Comment:
I actually had something like this originally, then realized it was calling constantly & I thought that I was seeing some choppiness as a result. But that's fine--we can go with this, and it'll have the benefit of updating more nicely when clicking around while paused.